keyboards/studiokestra/nue/readme.md

```diff
@@ -0,0 +1,13 @@
+# Nue
```

Comment:
Could you add instructions on how to jump to the bootloader, so the firmware can be flashed?